executor/src/witgen/machines/mod.rs

```diff
@@ -214,3 +215,34 @@ impl<'a, T: FieldElement> MachineParts<'a, T> {
         self.fixed_data.column_name(poly_id)
     }
 }
+
+/// The minimum size for which a warning is logged if the used rows are less than half of the size.
+/// This number coincides with 2**powdr_linker::MIN_DEGREE_LOG.
```

Comment:
It doesn't because there is no static_assert.

Reply:
Is it important that it does?

Reply:
It is not *really* important. If the total number of rows, the overhead of proving them becomes negligible and the log would just be noise. Arguably 32 is a already very small.
If this number was below `2**powdr_linker::MIN_DEGREE_LOG`, we'd get the log message often, that's why I chose it to be equal.

Yesterday, we also discussed with Thibaut and Leo that we want to remove `MIN_DEGREE_LOG` and let the user always be explicit about the min degree. Then, this comment goes away.